### PR TITLE
PGOV-794 Upgrading Cloudy to Bootstrap 5

### PR DESCRIPTION
**Pull request by odensc**

### CODE REVIEW

**Review by joshuami — DISMISSED** [review]

File: web/sites/default/config/views.view.admin_park_facilities.yml [hunk 1]
```diff
@@ -562,7 +562,7 @@ display:
...
+            text: "{% if moderation_state == \"Published\" %}\r\n<span class=\"badge bg-success\" title=\"{{ moderation_state }}\" data-bs-toggle=\"tooltip\">{{ moderation_state }}</span>\r\n{% elseif moderation_state == \"In review\" %}\r\n<span class=\"badge bg-warning\" title=\"{{ moderation_state }}\" data-bs-toggle=\"tooltip\">{{ moderation_state }}</span>\r\n{{ field_reviewer }}\r\n{% elseif moderation_state == \"Unpublished/Archived\" %}\r\n<span class=\"badge bg-danger\" title=\"{{ moderation_state }}\" data-bs-toggle=\"tooltip\">{{ moderation_state }}</span>\r\n{% elseif moderation_state == \"Draft\" %}\r\n<span class=\"badge bg-warning\" title=\"{{ moderation_state }}\" data-bs-toggle=\"tooltip\">{{ moderation_state }}</span>\r\n{% else %}\r\n<span class=\"badge bg-light\" title=\"{{ moderation_state }}\" data-bs-toggle=\"tooltip\">{{ moderation_state }}</span>\r\n{% endif %}\r\n{{ is_locked }}"
```
[the remark this answers is not in the document]
Comment (odensc) [reply web/sites/default/config/views.view.admin_park_facilities.yml:565]: I noticed that but didn't dig too deep. That make senses, I'd assume it happened when the admin theme was updated a while back? I can remove these field rewrites to clean up the views.

File: web/modules/custom/portland/portland.module [hunk 2]
```diff
@@ -256,7 +256,7 @@ function portland_preprocess_menu(&$vars)
           unset($vars['items'][$key]);
           return;
         }
-        $item['title'] = Drupal\Core\Render\Markup::create('Alerts <span class="badge badge-dark">' . $alertCount . '</span>');
+        $item['title'] = Drupal\Core\Render\Markup::create('Alerts <span class="badge bg-dark">' . $alertCount . '</span>');
```
[the remark this answers is not in the document]
Comment (odensc) [reply web/modules/custom/portland/portland.module:259]: You're right, I think it may be a relic from the past :)

image

I can remove this and related code.

File: composer.json [hunk 3]
```diff
@@ -79,6 +79,7 @@
         "drupal/group": "^1.4",
         "drupal/groupmedia": "^2.0@alpha",
         "drupal/honeypot": "^2.0",
+        "drupal/hreflang": "^1.10",
```
Comment (RichardDavies) [inline composer.json:82]: Why is this change included in this PR? The addition of hreflang belongs to PGOV-743 whose commits aren't even in this branch.

Comment (odensc) [reply composer.json]: @RichardDavies I merged in master and those weird changes are gone now. I think I somehow merged in the commits from master without updating the head of my branch, so GitHub thought they were part of my PR. 🤷

**Review by RichardDavies — APPROVED** [review]
